tests/decodeinputavformat.cpp

```diff
@@ -38,11 +38,11 @@ bool DecodeInputAvFormat::initInput(const char* fileName)
         goto error;
     uint32_t i;
     for (i = 0; i < m_format->nb_streams; ++i) {
-        AVCodecContext* ctx = m_format->streams[i]->codec;
```

Comment:
IMO, this requires more discussion as which libavcodec version we will support. Once we agree on that I'll change this patch.